libraries-testing/pom.xml

```diff
@@ -193,7 +200,7 @@
 
     <properties>
         <asciidoctor.version>1.5.7.1</asciidoctor.version>
```

Comment:
This property, and the following related to serenity seems to be particular of your environment. If so please remove them.

Reply:
In this case, all I did was to fix the identation, which was using tabs, to the recommended number of spaces. All other properties were already there - except, of course, the ArchUnit's

Reply:
Ok. Thanks